### PR TITLE
Dark mode enabled | Navbar modifications | Mobile menu modifications

### PR DESCRIPTION
**Pull request by tanishq-xin**
image

image

### CODE REVIEW

File: app/layout.tsx
Comment (Ellahinator) [inline app/layout.tsx:1]: `<PHProvider>` should go outside the `body`

File: .hintrc
Comment (ICEPrey) [inline .hintrc:1]: remove this file since we already have a .eslintrc config file

File: app/globals.css [hunk 1]
```diff
@@ -2,22 +2,52 @@
 @tailwind components;
 @tailwind utilities;
 
+@import url("https://fonts.googleapis.com/css2?family=Figtree:ital,wght@0,300..900;1,300..900&display=swap");
+
+* {
+  font-family: "Figtree", sans-serif !important;
+}
+
```
Comment (ICEPrey) [inline app/globals.css:10]:
```suggestion
```

Remove this, throwing error + pan didn't decide on a font yet

File: components/auth/signup.tsx [hunk 2]
```diff
@@ -98,33 +98,38 @@ export default function SignUp() {
...
+                      className="border-[white] mr-4 flex h-6 items-center border-r border-opacity-50 dark:border-opacity-40"
                       aria-hidden="true"
-                    />
-                    <span className="-ml-16 flex-auto pl-16 pr-8">
+                    /> */}
+                    <span className="">
+                      {" "}
+                      {/* -ml-16 pl-16 pr-8 flex-auto */}
                       Sign up with GitHub
                     </span>
```
Comment (ICEPrey) [inline components/auth/signup.tsx:134]:
```suggestion
                    <span>Sign up with GitHub</span>
```

File: components/auth/signup.tsx [hunk 3]
```diff
@@ -98,33 +98,38 @@ export default function SignUp() {
...
-                    className="relative flex w-full items-center rounded-md bg-gray-700 px-0 text-white-main hover:bg-gray-800"
+                    variant="authgroup"
+                    className="relative flex w-full items-center rounded-md px-0"
                   >
-                    <GitHubLogo className="text-white mx-4 h-4 w-4 shrink-0" />
-                    <span
-                      className="border-white mr-4 flex h-6 items-center border-r border-opacity-25"
+                    <GitHubLogo className="mx-1 h-4 w-4 shrink-0 text-gray-700" />
+                    {/* <span
+                      className="border-[white] mr-4 flex h-6 items-center border-r border-opacity-50 dark:border-opacity-40"
```
Comment (ICEPrey) [inline components/auth/signup.tsx:127]: remove comments/not used code

File: components/auth/signup.tsx [hunk 4]
```diff
@@ -98,33 +98,38 @@ export default function SignUp() {
...
+                  className="border-[white] mr-4 flex h-6 items-center border-r border-opacity-50 dark:border-opacity-40"
                   aria-hidden="true"
-                />
-                <span className="-ml-16 flex-auto pl-16 pr-8">
+                /> */}
+                <span className="">
+                  {" "}
+                  {/* -ml-16 pl-16 pr-8 flex-auto */}
                   Sign up with Google
                 </span>
```
Comment (ICEPrey) [inline components/auth/signup.tsx:113]:
```suggestion
                <span>Sign up with Google</span>
```

File: components/auth/signup.tsx [hunk 5]
```diff
@@ -98,33 +98,38 @@ export default function SignUp() {
...
                 size="lg"
-                variant="danger"
+                variant="authgroup"
                 className="relative flex w-full items-center rounded-md px-0"
               >
-                <GoogleLogo className="text-white mx-4 h-4 w-4 shrink-0" />
-                <span
-                  className="border-white mr-4 flex h-6 items-center border-r border-opacity-25"
+                <GoogleLogoColored className="text-white mx-1 h-4 w-4 shrink-0" />
+                {/* <span
```
Comment (ICEPrey) [inline components/auth/signup.tsx:105]: also remove comments/unused code here

File: public/nka.png
Comment (ICEPrey) [inline public/nka.png:1]: What is this? lol

File: components/auth/signin.tsx [hunk 6]
```diff
@@ -78,33 +80,38 @@ export default function SignIn() {
...
+                  className="border-[white] mr-4 flex h-6 items-center border-r border-opacity-50 dark:border-opacity-40"
                   aria-hidden="true"
-                />
-                <span className="-ml-16 flex-auto pl-16 pr-8">
+                /> */}
+                <span className="">
+                  {" "}
+                  {/* -ml-16 pl-16 pr-8 flex-auto */}
                   Sign in with Google
                 </span>
```
Comment (ICEPrey) [inline components/auth/signin.tsx:95]:
```suggestion               
 <span>Sign up with Google</span>
```

File: components/auth/signin.tsx [hunk 7]
```diff
@@ -78,33 +80,38 @@ export default function SignIn() {
...
-                <GoogleLogo className="text-white mx-4 h-4 w-4 shrink-0" />
-                <span
-                  className="border-white mr-4 flex h-6 items-center border-r border-opacity-25"
+                <GoogleLogoColored className="text-white mx-1 h-4 w-4 shrink-0" />
+                {/* <span
+                  className="border-[white] mr-4 flex h-6 items-center border-r border-opacity-50 dark:border-opacity-40"
                   aria-hidden="true"
-                />
-                <span className="-ml-16 flex-auto pl-16 pr-8">
+                /> */}
```
Comment (ICEPrey) [inline components/auth/signin.tsx:90]: remove comment

File: components/auth/signin.tsx [hunk 8]
```diff
@@ -78,33 +80,38 @@ export default function SignIn() {
...
-                    <GitHubLogo className="text-white mx-4 h-4 w-4 shrink-0" />
-                    <span
-                      className="border-white mr-4 flex h-6 items-center border-r border-opacity-25"
+                    <GitHubLogo className="mx-1 h-4 w-4 shrink-0 text-gray-700" />
+                    {/* <span
+                      className="border-[white] mr-4 flex h-6 items-center border-r border-opacity-50 dark:border-opacity-40"
                       aria-hidden="true"
-                    />
-                    <span className="-ml-16 flex-auto pl-16 pr-8">
+                    /> */}
```
Comment (ICEPrey) [inline components/auth/signin.tsx:111]: remove comment

File: components/auth/signin.tsx [hunk 9]
```diff
@@ -78,33 +80,38 @@ export default function SignIn() {
...
+                    <span className="">
+                      {" "}
+                      {/* -ml-16 pl-16 pr-8 flex-auto */}
                       Sign in with GitHub
                     </span>
```
Comment (ICEPrey) [inline components/auth/signin.tsx:116]:
```suggestion
                    <span>Sign in with GitHub</span>
```

File: components/about.tsx [hunk 10]
```diff
@@ -26,51 +26,68 @@ const videoData: VideoData[] = [
...
-          development, and we believe that with the advancement of AI, it may
-          drastically change over the coming years. We aim to build the
-          environment that will encompass these changes, both in the short-term
-          and long-term.
-        </p>
-      </div>
-      <div className="mt-10 flex flex-col items-center justify-center">
-        <p className="text-xs text-gray-500">Current features include</p>
+    <section
+      className="to-white bg-gradient-to-b from-gray-50 px-4 py-12 pt-20 dark:bg-none"
```
Comment (ICEPrey) [inline components/about.tsx:30]:
```suggestion
      className="to-white bg-gradient-to-b from-gray-50 px-4 py-12 pt-20"
```

File: components/ui/authbutton.tsx [hunk 11]
```diff
@@ -15,37 +15,37 @@ export default async function AuthButton() {
...
-    <div className="hidden flex-row items-center space-x-1 text-secondary-600 md:flex">
+    <div className="hidden flex-row items-center space-x-1 md:flex">
       {error || !data?.user ? (
         <div className="flex flex-row items-center space-x-1">
-          <CircleUserRound className="h-5 w-5" />
+          <CircleUserRound strokeWidth={1} className="h-5 w-5" />
 
           <Link
-            className="transition duration-150 ease-in-out hover:text-secondary-400"
+            className="transition duration-150 ease-in-out hover:text-gray-700"
```
Comment (ICEPrey) [inline components/ui/authbutton.tsx:24]:
```suggestion
            className="text-gray-800 transition duration-150 ease-in-out hover:text-primary-800"
```

File: components/ui/authbutton.tsx [hunk 12]
```diff
@@ -15,37 +15,37 @@ export default async function AuthButton() {
...
             href={"/signin"}
           >
             Sign in
           </Link>
-          <span className="text-secondary-600">/</span>
+          <span className="text-gray-400 opacity-50">/</span>
           <Link
-            className="transition duration-150 ease-in-out hover:text-secondary-400"
+            className="transition duration-150 ease-in-out hover:text-gray-700"
```
Comment (ICEPrey) [inline components/ui/authbutton.tsx:31]:
```suggestion
            className="text-gray-800 transition duration-150 ease-in-out hover:text-primary-800"
```

File: components/ui/authbutton.tsx [hunk 13]
```diff
@@ -15,37 +15,37 @@ export default async function AuthButton() {
...
             Sign up
           </Link>
         </div>
       ) : (
         <>
-          <CircleUserRound className="h-5 w-5" />
+          <CircleUserRound strokeWidth={1} className="h-5 w-5" />
           <Link
-            className="transition duration-150 ease-in-out hover:text-secondary-400"
+            className="transition duration-150 ease-in-out hover:text-gray-700"
```
Comment (ICEPrey) [inline components/ui/authbutton.tsx:41]:
```suggestion
            className="text-gray-800 transition duration-150 ease-in-out hover:text-primary-800"

```

File: components/ui/authbutton.tsx [hunk 14]
```diff
@@ -15,37 +15,37 @@ export default async function AuthButton() {
...
             href={"/settings"}
           >
             Settings
           </Link>
-          <span className="text-secondary-600">/</span>
+          <span className="text-gray-400 opacity-50">/</span>
           <form action={handleSignOut}>
-            <button className="transition duration-150 ease-in-out hover:text-secondary-400">
+            <button className="transition duration-150 ease-in-out hover:text-gray-700">
```
Comment (ICEPrey) [inline components/ui/authbutton.tsx:48]:
```suggestion
            <button className="text-gray-800 transition duration-150 ease-in-out hover:text-primary-800">
```

File: components/ui/header.tsx [hunk 15]
```diff
@@ -42,53 +45,65 @@ export default async function Header() {
...
+                      key={link.label}
                     >
-                      {link.label}
-                    </Link>
-                  </li>
-                ))}
-              </ul>
-            </nav>
+                      <Link
+                        className="align-middle hover:text-gray-700"
```
Comment (ICEPrey) [inline components/ui/header.tsx:67]:
```suggestion
                        className="align-middle text-gray-800 transition duration-150 ease-in-out hover:text-primary-800"
```

**Review by ICEPrey — CHANGES_REQUESTED** [review]
couple of changes

File: public/nka.png
Comment (tanishq-xin) [reply public/nka.png:1]: whoops must have got it here by mistake

**Review by Fryingpannn — COMMENTED** [review]
- The about page was updated (see https://trypear.ai/about). Needs some fixing in this PR 👀. If too much to merge the conflicts correctly, may be easier to directly copy the current file and simply add dark mode to it.
- Mobile responsiveness for the bottom buttons
Screenshot 2024-07-14 at 3 45 58 PM
Screenshot 2024-07-14 at 3 46 15 PM

**Review by Fryingpannn — APPROVED** [review]

**Pull request merged**